### PR TITLE
US 1583733 Add missing language IDs - 10

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "output" for program output examples.
- Used "console" for csc command line C# compiler examples and snk (strong name tool) commands.
- Used "env" for Environment variable settings.
- Used "text" for examples of assembly reference strings in XML metadata.

Contributes to #2192

### CODE REVIEW

**Comment by mairaw** [comment]
This looks good @v-thpra. I'm putting this one on hold just because it conflicts with a huge PR we have going on for content under the app-domains folder that I'm trying to avoid creating merge conflicts. So we'll deal with the merge conflicts here later on.

**Review by BillWagner — COMMENTED** [review]
I've reviewed the changes, and everything LGTM.

Once the related PR is merged, and we address any merge conflicts, I'll give it a final review and ship it.

**Review by mairaw — CHANGES_REQUESTED** [review]
@v-thpra now that the other PR was merged, we have several conflicts. Please rebase your branch. Thanks!

**Comment by v-thpra** [comment]
@mairaw The conflicts should be fixed now. Thanks.

**Comment by mairaw** [comment]
I've deleted some of the files that were being reincluded. They were moved to a new location instead.

**Comment by tfosmark** [comment]
@v-thpra Please make sure Maira's requested changes are marked as Resolved. Thanks!

**Comment by v-thpra** [comment]
@mairaw I believe all of the changes have been addressed in this PR. It should be ready to merge. Thanks!

File: docs/csharp/tour-of-csharp/program-structure.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ Because an assembly is a self-describing unit of functionality containing both c
...
 
-```
+```console
 csc /r:acme.dll example.cs
 ```
 
 This creates an executable assembly named `example.exe`, which, when run, produces the output:
 
-```
+```output
```
Comment (mairaw) [inline docs/csharp/tour-of-csharp/program-structure.md]:
```suggestion
```console
```

File: docs/csharp/versioning.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Take the following example:
 
 **Output**
 
-```
+```output
```
Comment (mairaw) [inline docs/csharp/versioning.md]:
```suggestion
```console
```

File: docs/csharp/versioning.md [hunk 3]
```diff
@@ -113,7 +113,7 @@ hides it. The base class member needs to have the `virtual` modifier applied to
...
```
Comment (mairaw) [inline docs/csharp/versioning.md]:
```suggestion
```console
```

File: docs/standard/assembly/enhanced-strong-naming.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ A strong name signature is an identity mechanism in the .NET Framework for ident
 
     This output can then be transformed into an AssemblySignatureKeyAttribute.
 
-    ```
+    ```text
```
Comment (mairaw) [inline docs/standard/assembly/enhanced-strong-naming.md]: Let's remove this identifier

File: docs/standard/assembly/names.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ An assembly's name is stored in metadata and has a significant impact on the ass
   
  The runtime uses this information to locate the assembly and differentiate it from other assemblies with the same name. For example, a strong-named assembly called `myTypes` could have the following fully qualified name:  
   
-```  
+```text
```
Comment (mairaw) [inline docs/standard/assembly/names.md]: let's not add an identifier to these two blocks in this article

**Review by mairaw — COMMENTED** [review]
Thank you @v-thpra. Left you a few other comments.

File: docs/standard/assembly/enhanced-strong-naming.md (hunk 4 above)
Comment (v-thpra) [reply docs/standard/assembly/enhanced-strong-naming.md]: Fixed

**Comment by v-thpra** [comment]
@mairaw I Implemented your suggestions. Thanks!

**Review by mairaw — APPROVED** [review]
Thank you @v-thpra!

**Pull request merged**